Improbable/DatabaseSync/Improbable.DatabaseSync.Schema/Improbable.DatabaseSync.Schema.csproj

```diff
@@ -1,16 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-
     <IncludeBuildOutput>false</IncludeBuildOutput>
     <Version>0.0.2-preview</Version>
```

Comment:
As a point of order, these versions are desynced with the git release tags - are we going to bring these back into sync?

Reply:
And also seems out of sync within this package too - https://github.com/spatialos/csharp-worker-template/pull/15/files#diff-730dea55b4430d620669d524d1dc90ddR3

Reply:
I've been trying out incrementing the version tags only when a particular package changes, rather than changing them as one. Though yes, they should all be incremented in this PR!

Reply:
> I've been trying out incrementing the version tags only when a particular package changes, rather than changing them as one. 

I think this makes more sense if we were releasing them independently, since for now we are releasing in "lockstep", it seems like less overhead to version them together. But it's not a huge amount of overhead so 🤷‍♂